automation/vars/hci.yaml

```diff
@@ -76,3 +76,78 @@ vas:
           - name: edpm-deployment-values-post-ceph
             src_file: values.yaml
         build_output: deployment-post-ceph.yaml
+
+# Adoption related content for adoption
+adoption:
+  hci:
+    layout_patch:
```

Comment:
Just diving a bit deeper into CI framework (this is my first real encounter with it).

If this is patching `cifmw_libvirt_manager_layout` then should we name the patch variables accordingly? Like `cifmw_libvirt_manager_layout_patch`? I wouldn't be afraid of long names for the sake of clarity.

Also where does the actual `cifmw_libvirt_manager_layout` value come from for the architecture jobs? I was wondering about the patching approach in general, vs. just redefining the variable for adoption, but i can't find the variable for greenfield here in this repo or in ci-framework. Is there another repo i should be looking at?

Reply:
FWIW i was looking at https://github.com/openstack-k8s-operators/ci-framework/pull/2285/files#diff-8599ae825f8eabc23397e4f5dd09e71bd037657b1f3e837565cb162d245b3a1cR93 . I guess `_cifmw_libvirt_manager_layout` there gets processed from `cifmw_libvirt_manager_configuration`? https://github.com/openstack-k8s-operators/ci-framework/blob/b349f2735319daea1d1b43cb6b14cd7af4fa16db/scenarios/reproducers/va-hci.yml#L21

Reply:
The patching is done here:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/libvirt_manager/tasks/generate_layout.yml

patches are nice, since we don't need to redefine the whole thing. There's one weakness though : we can't delete any entry.

The playbooks I proposed in https://github.com/openstack-k8s-operators/ci-framework/pull/2285 are taking care of the dataset manipulation/patching